__tests__/helpers/on-headers-listener.spec.js

```diff
@@ -7,7 +7,8 @@ describe('helpers', () => {
       const spans = defaultConfig.spans;
 
       beforeEach(() => {
-        spans.forEach((span) => {
+        spans.forEach((currentSpan) => {
```

Comment:
should have a different meaning - maybe make spans a let and use map to iterate over the spans and clear the responses:

```javascript
let spans = defaultConfig.spans;
// ....
spans = spans.map((currentSpan) => {
  const span = currentSpan;
  span.responses = [];
  return span;
});
// ...